app/core/components/StickyHeaderTable.module.css

```diff
@@ -0,0 +1,10 @@
+.box {
+  overflow-y: scroll !important;
```

Comment:
Use `!important` because maybe chakra UI set the property `overflow-y: hidden` to the `.box` 🤔